.gitignore

```diff
@@ -21,3 +21,6 @@ dev/node_modules
 # Dependency directories (remove the comment below to include it)
```

Comment:
You've created multiple commits, with changes that will be reverted in the next. Could you squash the commits into 2 commits? Splitting the changes inside the .gitignore and the addition of the auth feature?


* Add .idea to .gitignore
* Add authentication to /session

Reply:
I think I have spilt them.